README.md

```diff
@@ -49,10 +49,36 @@ for item in result.fetchall():
     print(item)
 ```
 
+### [AsyncIO](https://docs.sqlalchemy.org/en/14/orm/extensions/asyncio.html) extension
+
+```python
+from sqlalchemy import text
+from sqlalchemy.ext.asyncio import create_async_engine
+from firebolt_db.firebolt_async_dialect import AsyncFireboltDialect
+from sqlalchemy.dialects import registry
+
+registry.register("firebolt", "src.firebolt_db.firebolt_async_dialect", "AsyncFireboltDialect")
+engine = create_async_engine("firebolt://email@domain:password@sample_database/sample_engine")
+
+async with engine.begin() as conn:
```

Comment:
What's the difference between two?

Reply:
Between `engine.begin()` and `engine.connect()`? I believe the former also creates a transaction behind the scenes. Since we don't have transaction support it's equivalent to `engine.connect()`. It's probably confusing, I'll remove it from a readme.